doc/troubleshooting/howtoset_sel.md

```diff
@@ -0,0 +1,15 @@
+# How to set sel ?
+
+The setting of "sel" is related to "rcut" and the coordination number of certain atoms. Some empirical settings on some specific systems are listed below.
+
+
+system | rcut | sel
+---|---|---
+Li | 9.0 | [700]
+Li | 6.0 | [200]
+Si | 6.0 | [300]
+water | 6.0 | [200,400]
```

Comment:
Where does these recommendations come from?  I find the `sel` for water is too large for ambient conditions.

Reply:
I found this in your work directory on Aliyun. It might be a from a previous test, so may not be the final optimal setting.

Reply:
I rewrote this. Is it better now?

Reply:
The "empirical settings" are misleading and should be removed.

We should provide the users a practical guide of how to set the `sel` list.

Reply:
Removed.